src/machines/settingsMachine.ts

```diff
@@ -89,6 +91,13 @@ export const settingsMachine = setup({
         currentTheme === Themes.System ? getSystemTheme() : currentTheme
       )
     },
+    setEngineCameraProjection: () => {
+      if (sceneInfra.camControls.reactCameraProperties.type === 'perspective') {
+        sceneInfra.camControls.useOrthographicCamera()
+      } else {
+        sceneInfra.camControls.usePerspectiveCamera(true).catch(reportRejection)
+      }
```

Comment:
I feel this is the second time I've seen an if / else of this kind for determining the camera. It means this pattern is somewhat common. Might want to put it behind a function